src/inference/src/dev/core_impl_ie.cpp

```diff
@@ -267,10 +267,20 @@ std::map<std::string, InferenceEngine::Version> ov::CoreImpl::GetVersions(const
         ov::DeviceIDParser parser(deviceName_);
         std::string deviceNameLocal = parser.get_device_name();
 
-        ov::Plugin cppPlugin = get_plugin(deviceNameLocal);
-
-        versions[deviceNameLocal] =
-            ov::legacy_convert::convert_plugin(ov::SoPtr<ov::IPlugin>{cppPlugin.m_ptr, cppPlugin.m_so})->GetVersion();
+        try {
+            ov::Plugin cppPlugin = get_plugin(deviceNameLocal);
+            auto convertedPlugin =
+                ov::legacy_convert::convert_plugin(ov::SoPtr<ov::IPlugin>{cppPlugin.m_ptr, cppPlugin.m_so});
+            versions[deviceNameLocal] = convertedPlugin->GetVersion();
+        } catch (const ov::Exception& ex) {
+            std::string exception(ex.what());
+            if (exception.find("not registered in the OpenVINO Runtime") == std::string::npos) {
+                OPENVINO_THROW(ex.what());
+            }
+            InferenceEngine::Version version;
+            version.description = NULL;
+            versions[deviceNameLocal] = version;
```

Comment:
why do we need to add empty version? can we just skip such unregistered device?

Reply:
Adding empty version here is to provide the prompt message to user that plugin is not registered.

Reply:
we don't need to provide _debug capabilities_ here IMO
CC @ilyachur

Reply:
Prompt message is removed here.